x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -27,7 +27,7 @@ import './alerts_table.scss';
 export const ACTIVE_ROW_CLASS = 'alertsTableActiveRow';
 const AlertsFlyout = lazy(() => import('./alerts_flyout'));
 const GridStyles: EuiDataGridStyle = {
-  border: 'horizontal',
+  border: 'none',
```

Comment:
Design people -> when I added the fontSize attribute then the virtualization was not working anymore. That's why I added in the `.scss` file. any help/suggestion to avoid that is appreciated, thanks in advance.

Reply:
@chandlerprall I think whatever is going on here might be related to https://github.com/elastic/kibana/issues/132901